### PR TITLE
To do list 

### PR DESCRIPTION
**Pull request by King-Benx**
**Delivers the following:**
1. Implements webpack
2. Implements linters
3. Loads list of books dynamically.
4. Sorts list of books based on the index

**How to run:**
1. Clone the project ```git clone git@github.com:King-Benx/todo-list-es6.git```
2. Checkout to the root directory ```cd todo-list-es6```
3. Install dependencies ```npm i```
4. Run the project using webpack ```npm run dev```

**View online**
[https://king-benx.github.io/todo-list-es6/](https://king-benx.github.io/todo-list-es6/)

**Screenshots**
Screenshot 2023-02-24 at 9 51 57 AM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,148 @@
...
+
+```git clone git@github.com:King-Benx/todo-list.git```
+
+### Usage
+
+To run the project, execute the following command:
+
+```
+1. Right click index.html file and run in browser.
+2. Open index file in vscode right click and open in live server
```
Comment (Shedrack-Sunday) [inline README.md]: - [ ] Nice work on this section. however, you didn't include how to create and build your app using the webpack on your project. Kindly add the steps to create a build for your project

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @King-Benx   👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 



Highlights
✅All linters are checked
✅ Proper PR summary 
  
# Required Changes ♻️

- [x] I must admit you have really put in the effort so far in this project, however, you didn't build the `dist` folder which contains the webspace-generated HTML file. kindly create the build and include the steps on the README

_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @King-Benx,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

 - [ ] Kindly build the dist folder requested the previous reviewer.

 _Check the comments under the review for more changes_.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by TadesseDev — APPROVED** [review]
### Hi @King-Benx 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [ ] Good job making the changes requested by the previous code reviewer. ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**